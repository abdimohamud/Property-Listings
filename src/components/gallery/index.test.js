import * as React from 'react'
import renderer from 'react-test-renderer'

import Gallery from '.'

test('renders correctly', () => {
  const image = 'image.png'
  const title = 'title'
  const tree = renderer.create(<Gallery image={image} title={title} />).toJSON()
  expect(tree).toMatchSnapshot()
})
