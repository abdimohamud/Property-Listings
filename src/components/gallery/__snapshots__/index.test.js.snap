// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<figure
  className="figure"
>
  <img
    alt="title"
    className="img-responsive"
    src="/server/image.png"
  />
  <figcaption
    className="figure-caption text-center text-small"
  >
    title
  </figcaption>
</figure>
`;
